app/views/includes/yourTaxGraph.scala.html

```diff
@@ -14,16 +14,25 @@
  * limitations under the License.
  *@
 
-@(graph: uk.gov.hmrc.tai.viewModels.BandedGraph, taxRegion: String)(implicit request: Request[_], messages: Messages)
+@import uk.gov.hmrc.tai.viewModels.{Swatch,ZeroTaxView,TaxViewType}
 @import uk.gov.hmrc.play.views.helpers.MoneyPounds
 @import uk.gov.hmrc.play.views.formatting.Money._
 @import uk.gov.hmrc.tai.util.TaxRegionConstants._
-    <h3 class="heading-medium">@Messages("tai.estimatedIncomeTax.howWeWorkedOut")</h3>
+@import uk.gov.hmrc.tai.util.ViewModelHelper.{withPoundPrefix}
 
-    <div aria-hidden="true">
+@(graph: uk.gov.hmrc.tai.viewModels.BandedGraph, taxRegion: String, taxViewType:TaxViewType,incomeEstimate:BigDecimal)(implicit request: Request[_], messages: Messages)
+
+@swatchLeftPadding=@{ graph.swatch.foldLeft(BigDecimal(100))((acc: BigDecimal, next: Swatch) => acc - next.barPercentage)}
+
+
+    <div aria-hidden="true" class="number-bar__container">
         <div class="number-bar">
             <span style="left:0">@pounds(0)</span>
-            <span id="nextHigherBand" style="right:0">@pounds(graph.nextBand)</span>
```

Comment:
Since this is removed, then can the data in the view model be removed?
https://github.com/hmrc/tai-frontend/pull/112/files?utf8=✓&diff=split#diff-da2d182beb543630d927b4bfb29a3bbeL25